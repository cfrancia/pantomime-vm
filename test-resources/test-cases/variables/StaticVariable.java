/**
 * Created by cfrancia on 24/09/16.
 */
public class StaticVariable {

    private static int STATIC_INT = 2;

    public static void main(String[] args) {
        println(STATIC_INT);
        STATIC_INT = 3;
        println(STATIC_INT);
    }

    public static native void println(int val);

}
