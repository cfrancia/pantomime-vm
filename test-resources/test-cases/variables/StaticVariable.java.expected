OUT: 2
OUT: 3
